examples/rootca.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This also shouldn't be committed.

Reply:
This one's used in tests for the new feature and so is necessary.